scripts/qa_crowbarsetup.sh

```diff
@@ -3698,6 +3698,9 @@ function oncontroller_testsetup
 {
     . .openrc
     oncontroller_prepare_functional_tests
+    if [[ $hacloud = 1 ]] ; then
+        crm_mon --failcounts -1 | grep "fail-count=" && complain 34 "Cluster resources' failures detected"
```

Comment:
@aplanas, @jsuchome, @vmoravec 
The exit codes are not meant to be unique (we don't have enough to make them unique), just seldom. If you print a message with complain anyway, you can quickly find the error.
If its done via safely you need to check the command on the previous line(s).